test/integration/check_roles_enabled_test.lua

```diff
@@ -0,0 +1,72 @@
+local t = require('luatest')
+local g = t.group('check_roles_enabled_integration')
+local fiber = require('fiber') -- luacheck: ignore
+
+local fio = require('fio')
+
+local cartridge_helpers = require('cartridge.test-helpers')
+
+local shared = require('test.helper.integration').shared
+
+local datadir = fio.pathjoin(shared.datadir, 'basic')
+
+g.cluster = cartridge_helpers.Cluster:new({
+    server_command = fio.pathjoin(shared.root, 'test', 'entrypoint', 'check_roles_enabled_init.lua'),
+    datadir = datadir,
+    use_vshard = true,
+    replicasets = {
+        {
+            alias = 'api',
+            uuid = cartridge_helpers.uuid('a'),
+            roles = { 'vshard-router', 'migrator' },
+            servers = { { alias='api-master', instance_uuid = cartridge_helpers.uuid('a', 1) } },
+        },
+        {
+            alias = 'storage-role1-role2',
+            uuid = cartridge_helpers.uuid('b'),
+            roles = { 'vshard-storage', 'cartridge.roles.role1', 'cartridge.roles.role2' },
+            servers = {
+                {
+                    alias = 'storage-role1-role2-master',
+                    instance_uuid = cartridge_helpers.uuid('b', 1),
+                    env = {TARANTOOL_HTTP_ENABLED = 'false'},
+                },
+                {
+                    alias = 'storage-role1-role2-replica',
+                    instance_uuid = cartridge_helpers.uuid('b', 2),
+                    env = {TARANTOOL_HTTP_ENABLED = 'false'},
+                },
+            },
+        },
+    },
+})
+
+g.before_all(function() g.cluster:start() end)
+g.after_all(function() g.cluster:stop() end)
+
+g.test_check = function()
+    t.assert(g.cluster:server('api-master'):eval([[return require('migrator.utils').check_roles_enabled({'vshard-router'})]]))
+    t.assert(g.cluster:server('api-master'):eval([[return require('migrator.utils').check_roles_enabled({'vshard-router', 'migrator'})]]))
+    t.assert(g.cluster:server('api-master'):eval(
```

Comment:
```suggestion
g.before_all(function(cg) cg.cluster:start() end)
g.after_all(function(cg) cg.cluster:stop() end)

g.test_check = function(cg)
    t.assert(cg.cluster:server('api-master'):eval([[return require('migrator.utils').check_roles_enabled({'vshard-router'})]]))
    t.assert(cg.cluster:server('api-master'):eval([[return require('migrator.utils').check_roles_enabled({'vshard-router', 'migrator'})]]))
    t.assert(cg.cluster:server('api-master'):eval(
```

and below.

Current implementation will fail if group will became parametrized.